File: pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float n;
    do
    {
        n = get_float("Changed owed: "); //prompts the user for an amount of change
    }
    while (n < 0);
    int cents = round(n * 100);
    int quarters = cents / 25;
    int quarters_remainder = cents % 25;
    int dimes = quarters_remainder / 10;
    int dimes_remainder = quarters_remainder % 10;
    int nickels = dimes_remainder / 5;
    int pennies = dimes_remainder % 5;
    int coins = quarters + dimes + nickels + pennies;
    printf("%i\n", coins);
}

File: pset6/cash.py
from cs50 import get_float

change_owed = get_float("Change owed: ")

while change_owed < 0:
    change_owed = get_float("Change owed: ")

quarters = change_owed // 0.25
quarter_remainder = round(change_owed % 0.25, 2)

dimes = quarter_remainder // 0.10
dimes_remainder = round(quarter_remainder % 0.10, 2)

nickels = dimes_remainder // 0.05
nickels_remainder = round(dimes_remainder % 0.05, 2)

pennies = nickels_remainder // 0.01

coins = int(quarters + dimes + nickels + pennies)

print(coins)